LICENSES/ISC.txt

```diff
@@ -0,0 +1,14 @@
+ISC License Copyright (c) 2004-2010 by Internet Systems Consortium, Inc. ("ISC")
+
+Copyright (c) 1995-2003 by Internet Software Consortium
```

Comment:
Where does this specific version of the license come from?
Shouldn't the years also be omited here?

Reply:
It's what the reuse tool downloads automatically.

To simplify things, I've just used the version from iw instead.